ADMINS-FOR-ISTIO.md

```diff
@@ -6,21 +6,21 @@ access.
+| [Google Analytics](https://analytics.google.com/analytics/web/) | Analytics for *istio.io | [davidhauck](https://github.com/davidhauck)
+| [Google Calendar - Community Events](https://calendar.google.com/calendar/embed?src=i10ogf58krfbrsjai5qi16g4do%40group.calendar.google.com&ctz=America%2FLos_Angeles) | Community event scheduling | [macruzbar ](https://github.com/macruzbar)
 | [Google Calendar - Working Groups](https://calendar.google.com/calendar/embed?src=4uhe8fi8sf1e3tvmvh6vrq2dog%40group.calendar.google.com&ctz=America%2FLos_Angeles) | Working group scheduling | All working group leads
-| [Google Cloud DNS](https://cloud.google.com/dns) | DNS hosting for *istio.io | [jasonall](https://github.com/jasonall)
-| [Google Custom Search Engine](https://cse.google.com/cse/) | Search engine for *istio.io | [jasonall](https://github.com/jasonall), [rshriram](https://github.com/rshriram)
-| [Google Domains](https://domains.google.com) | Domain registration for *istio.io | [jasonall](https://github.com/jasonall)
+| [Google Cloud DNS](https://cloud.google.com/dns) | DNS hosting for *istio.io | [davidhauck](https://github.com/davidhauck)
+| [Google Custom Search Engine](https://cse.google.com/cse/) | Search engine for *istio.io | [davidhauck](https://github.com/davidhauck)
+| [Google Domains](https://domains.google.com) | Domain registration for *istio.io | [craigbox](https://github.com/craigbox)
 | [Google Groups](https://googlegroups.com) | Mailing lists | [craigbox](https://github.com/craigbox)
-| [Google Search Console](https://search.google.com/search-console/about) | Search engine analytics for *istio.io | [jasonall](https://github.com/jasonall)
-| [Netlify](https://netlify.com) | Site hosting for *istio.io | [jasonall](https://github.com/jasonall)
+| [Google Search Console](https://search.google.com/search-console/about) | Search engine analytics for *istio.io | [davidhauck](https://github.com/davidhauck)
+| [Netlify](https://netlify.com) | Site hosting for *istio.io | [davidhauck](https://github.com/davidhauck)
 | [Prow](https://prow.istio.io) | Primary CI system | [fejta](https://github.com/fetja),  [cjwagner](https://github.com/cjwagner)
-| [Shared Google Drive](https://drive.google.com/corp/drive/u/0/folders/0AIS5p3eW9BCtUk9PVA) | Design document storage | [rshriram](https://github.com/rshriram), [linsun](https://github.com/linsun), [louiscryan](https://github.com/louiscryan), [smawson](https://github.com/smawson), [jasminejaksic](https://github.com/jasminejaksic)
+| [Shared Google Drive](https://drive.google.com/corp/drive/u/0/folders/0AIS5p3eW9BCtUk9PVA) | Design document storage | [craigbox](https://github.com/craigbox), [linsun](https://github.com/linsun), [louiscryan](https://github.com/louiscryan), [smawson](https://github.com/smawson), [jasminejaksic](https://github.com/jasminejaksic)
```

Comment:
-jasmine

Reply:
@louiscryan I'm guessing we should remove `jasminejaksic` from this list. Can I be added to it and whatever needs to happen in the back end for it?

Reply:
Done, made all TOC members owners and fixed up Lin's broken email